Formula/ooniprobe.rb

```diff
@@ -200,41 +200,41 @@ def plist; <<-EOS.undent
    <dict>
      <key>Label</key>
        <string>#{plist_name}</string>
-     <key>Program</key>
-       <string>#{opt_bin}/ooniprobe</string>
+     <key>EnvironmentVariables</key>
+     <dict>
+       <key>PATH</key>
+       <string>#{HOMEBREW_PREFIX}/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
+     </dict>
      <key>ProgramArguments</key>
      <array>
-       <string>-i</string>
-       <string>#{pkgshare}/current.deck</string>
+       <string>#{opt_bin}/ooniprobe-agent</string>
+       <string>run</string>
      </array>
      <key>RunAtLoad</key>
-       <false/>
+       <true/>
      <key>KeepAlive</key>
-       <false/>
-     <key>StandardErrorPath</key>
-       <string>/dev/null</string>
-     <key>StandardOutPath</key>
-       <string>/dev/null</string>
```

Comment:
I was under the impression that not supplying a path would make the default be to not log, but I checked and it seems like this leads to them being logged.

I will add these back.